packages/maskbook/src/utils/hooks/useSuspense.ts

```diff
@@ -0,0 +1,27 @@
+import { useUpdate } from 'react-use'
+
+export function useSuspense<T, U extends Array<any>>(
```

Comment:
https://github.com/DimensionDev/Maskbook/pull/2827
Please check out this implementation. I'm afraid your implementation isn't correct.

Reply:
I would test #2827 later, this implementation works fine currently.

Reply:
no need to test #2827. it is not ready